backend/src/main/kotlin/be/osoc/team1/backend/entities/Student.kt

```diff
@@ -185,7 +186,8 @@ class Student(
     val communications: MutableList<Communication> = mutableListOf()
 
     @OneToMany(mappedBy = "student", cascade = [CascadeType.DETACH], orphanRemoval = true)
-    @JsonIgnore
+    @field:JsonView(StudentView.Full::class, StudentView.Extra::class)
```

Comment:
Can you remove view Full from the assignments? We don't actually ever use those on the full view & it might cause some issues frontend with showing the projects.